### PR TITLE
Update ansible to stable channel

### PR DESCRIPTION
**Pull request by fxiang1**
Signed-off-by: Feng Xiang <fxiang@redhat.com>

- Change name to Ansible automation platform as the resource operator is deprecated and removed
- Update ansible to stable channel
- Change ansible version

### CODE REVIEW

File: tests/cypress/templates/ansible_yaml/ansible-subscription.yaml [hunk 1]
```diff
@@ -1,12 +1,12 @@
 apiVersion: operators.coreos.com/v1alpha1
 kind: Subscription
 metadata:
-  name: awx-resource-operator
+  name: ansible-automation-platform-operator
```
Comment (chenz4027) [inline tests/cypress/templates/ansible_yaml/ansible-subscription.yaml:4]: If we change the name here, we may also need to change it in start-cypress-tests.sh as it deletes the operator with name

**Review by chenz4027 — CHANGES_REQUESTED** [review]
@fxiang1

File: tests/cypress/templates/ansible_yaml/ansible-subscription.yaml (hunk 1 above)
Comment (fxiang1) [reply tests/cypress/templates/ansible_yaml/ansible-subscription.yaml:4]: Thanks @chenz4027. That's a good catch. I made the change now.

**Comment by fxiang1** [comment]
@chenz4027 Can you review this again? The e2e test passed now. Thanks.

**Review by chenz4027 — APPROVED** [review]
lgtm!

**Pull request merged**